dev/test-studio/preview/loader.tsx

```diff
@@ -6,7 +6,7 @@ const client = createClient({
   projectId: 'ppsg7ml5',
   dataset: 'playground',
   useCdn: true,
-  apiVersion: '2023-02-06',
+  apiVersion: 'X',
```

Comment:
huh, just to double check, this is intended to be left in the corel branch?

Reply:
Yep, the preview iframe runs its own setup. It's similar to how applications running Content Releases will also have to specify vX is apps running live preview 🙌